jinahub/indexers/cache/Dockerfile

```diff
@@ -12,4 +12,4 @@ RUN pip install -r requirements.txt
 # for testing the image
 RUN pip install pytest && pytest -s -v tests
 
-ENTRYPOINT ["jina", "pod", "--uses", "config.yml"]
+ENTRYPOINT ["jina", "pea", "--uses", "config.yml"]
```

Comment:
I think the `FROM base` is still missing, right @jakob1111996 ?

Reply:
added